app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,6 +40,10 @@
     @Column(name = "operation_type")
     public String operation_type;
 
+    private static Context mContext = Home.getContext();
```

Comment:
Ok. Happy to do that. We need a style guide to ensure others don't make this mistake. 

 
John Stevens
M - +61 418526349

> On 20 May 2015, at 00:08, Stephen Black <notifications@github.com> wrote:
> 
> In app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java:
> 
> > @@ -39,6 +40,10 @@
> >      @Column(name = "operation_type")
> >      public String operation_type;
> >  
> > +    private static Context mContext = Home.getContext();
> Each time we enter this class we always come in through the same function and it already has a context passed in, we should not reach out to home to get its context, I changed this in the other branch already (this is what triggers those crashes)
> 
> —
> Reply to this email directly or view it on GitHub.
>